README.md

```diff
@@ -112,7 +112,9 @@ tag_object(pid, cid)
 - If desired, this cid can then be used to locate the object on disk by following HashStore's store configuration.
 
 **How do I delete an object if I have the pid?**
-- To delete an object, call the Public API method `delete_object` which will delete the object and its associated references and reference files where relevant.
+- To delete an object and all its associated reference files, call the Public API method `delete_object` with `id_type` 'pid'.
+- To delete only an object, call `delete_object` with `id_type` 'cid' which will remove the object if it it is not referenced by any pids.
+- To delete an object and all its related data (reference files and system metadata), call the Public API method `delete_object` with `id_type` 'clear'.
```

Comment:
>  with `id_type` 'clear'

this doesn't agree with what happens in the code

Reply:
Sorry, that statement/javadoc was not up to date. I've removed it.